vhost/src/vhost_user/message.rs

```diff
@@ -203,6 +205,12 @@ enum_value! {
         VRING_CALL = 4,
         /// Indicate that an error occurred on the specific vring.
         VRING_ERR = 5,
+        /// Add a virtio shared object.
+        SHARED_OBJECT_ADD = 6,
+        /// Remove a virtio shared object.
+        SHARED_OBJECT_REMOVE = 7,
+        /// Lookup for a virtio shared object.
+        SHARED_OBJECT_LOOKUP = 8,
```

Comment:
Should we also add `SHARED_OBJECT` in `VhostUserProtocolFeatures`?

And maybe check it while sending/receiving these messages.